Model/Product/Index/IndexRepository.php

```diff
@@ -99,6 +100,41 @@ public function getById($id)
         return $collection->getFirstItem(); // @codingStandardsIgnoreLine
     }
 
+    /**
+     * @inheritdoc
+     */
+    public function getTotalOutOfSync(Store $store = null)
+    {
+        /* @var IndexCollection $collection */
+        $collection = $this->indexCollectionFactory->create();
+        $collection->addFilter(
+            ProductIndexInterface::IN_SYNC,
+            NostoIndex::DB_VALUE_BOOLEAN_FALSE,
+            'eq'
+        );
+        if ($store) {
+            $collection->addStoreFilter($store);
+        }
+        return $collection->getSize();
```

Comment:
Does getSize do a query or a count?

Reply:
Queries `SELECT* COUNT ...`
whereas `$collection->count()` fetches and uses PHP count method.